### DISCUSSION TITLE
Estimating Outputs on further Timepoints

### DISCUSSION THREAD

**Discussion by ColeCampton** [nlmixr2/nlmixr2#161] [Q&A] [answered]
Hello, 

After fitting the estimated model has only the time points that existed in the original data set.  I'm curious what the proper means of estimating a fit on new timepoints is? 
```
est <- nlmixr2(model, data, est="saem")
interpolated_est <- interpolate(est, times = seq(from = 0, to = Tmax, by = 1))
```

After fitting of course I could extract the sample level estimations, and integrate the ODE with any package of my choice. This can be a bit tedious if making changes to the model itself.

**Comment by mattfidler** [comment, marked as answer]
Well the easiest is to use `augPred()` is the easiest.  

This comes from `nonmem2rx` but applies to any nlmixr2 fit:

https://nlmixr2.github.io/nonmem2rx/articles/create-augPred.html

**Reply by ColeCampton** [reply]
Thanks for the reference! I appreciate all of the work you've put into nlmixr!

**Reply by mattfidler** [reply]
No problem
